tests/blob-store-test/src/main.rs

```diff
@@ -0,0 +1,65 @@
+    Vivamus quis dictum ipsum. Proin congue vulputate elit rhoncus congue. Mauris elementum libero mollis, mattis nibh eu, commodo erat. Duis sodales feugiat diam, ut tempor purus viverra eu. Vestibulum sodales sit amet eros quis placerat. Phasellus venenatis condimentum nisl eget tristique. Suspendisse potenti. Cras a orci nec elit pulvinar condimentum at et diam. Mauris faucibus aliquam posuere. Donec accumsan, metus ac scelerisque iaculis, felis enim ullamcorper ante, quis tristique elit justo et ex. Donec et velit pharetra, viverra lacus vitae, laoreet elit.
+    Duis sodales odio velit, nec ornare leo venenatis ut. Pellentesque in libero sit amet libero vestibulum lobortis. Vivamus laoreet ex eget mi suscipit, vitae volutpat felis iaculis. Etiam rhoncus ac arcu nec commodo. Phasellus posuere, mi eget egestas tincidunt, orci tellus placerat turpis, sit amet blandit nibh magna eu est. Nunc ultrices tincidunt rhoncus. Praesent faucibus id augue quis laoreet. Maecenas ac pellentesque eros, id pellentesque magna. Integer faucibus auctor ante. Mauris vitae pharetra erat. Nulla facilisi. Fusce ac ex libero. Duis posuere lacus lectus, ut varius sapien efficitur eget. Nunc enim urna, congue non tristique id, consequat eget nisi. Nulla in massa sit amet nisi rhoncus tempor sagittis id erat.
+    Donec in accumsan odio. Integer faucibus velit posuere sem commodo tincidunt. Fusce facilisis ex eget nisl feugiat pulvinar. Curabitur dolor diam, tempus in ligula ut, feugiat cursus elit. Sed et neque molestie, vehicula diam sed, ultricies justo. Maecenas rutrum pharetra sapien eu interdum. Donec vulputate, massa quis malesuada eleifend, ex arcu viverra magna, sit amet volutpat sem risus sit amet dui. Sed ultricies at eros at consequat. Morbi fringilla tristique mauris vel iaculis. Pellentesque ornare dictum finibus. Fusce aliquet odio a blandit interdum.
+    Donec mollis finibus metus in cursus. In blandit ornare purus. Vestibulum a ipsum diam. Curabitur vel iaculis diam, id egestas ligula. Morbi condimentum imperdiet tellus. Aenean ut ligula nulla. Nam quis auctor odio. Sed eget blandit magna, sit amet consectetur ex. Quisque fermentum nunc at nisi dictum molestie. Ut tempus fermentum ipsum, vel aliquet sem rutrum quis. Nunc nec tristique diam.
+    Touch test.".as_bytes();
+    let bucket = blob_store::Container::open("slight-test-bucket")?;
+
+    // verify container name
+    assert_eq!(bucket.name()?, "slight-test-bucket");
+
+    for name in bucket.list_objects()? {
+        println!("Found object: {}", name);
+        bucket.delete_object(&name)?;
+    }
+
+    // upload 3 files
+    for i in 0..3 {
+        // let body = std::fs::read(format!("testfile{i}.txt")).expect("should have been able to read the file");
```

Comment:
delete this?

```suggestion
```

Reply:
Instead of deleting them, I will write a TODO comment saying that in the future, we want to enable this to allow users to upload files from local filesystem to blob stores. There are some security constraints with wasm modules, such as that they can't access random places in the filesystem, and thus we need to think throughly of how to allow them to upload files.

Reply:
Resovled